tests/test_daily_temperature_extrema.py

```diff
@@ -8,22 +8,28 @@
 from pycds.weather_anomaly import DailyMaxTemperature, DailyMinTemperature
 
 
-def describe_function_effective__day__for__Tmax():
-    @mark.parametrize('tod, obs_time', [
-        ('morning', '2000-01-01 07:23'),
-        ('afternoon', '2000-01-01 16:18')
-    ])
-    @mark.parametrize('freq, expected_day', [
-        ('1-hourly', {'morning': datetime.datetime(2000, 1, 1), 'afternoon': datetime.datetime(2000, 1, 1)}),
-        ('12-hourly', {'morning': datetime.datetime(2000, 1, 1), 'afternoon': datetime.datetime(2000, 1, 2)})
-    ])
-    def it_returns_the_expected_day_of_observation(mod_empty_database_session, tod, obs_time, freq, expected_day):
+def describe_function_effective__day():
+    expected_day = {
+        'max': {
+            '1-hourly': {'2000-01-01 07:23': datetime.datetime(2000, 1, 1), '2000-01-01 16:18': datetime.datetime(2000, 1, 1)},
+            '12-hourly': {'2000-01-01 07:23': datetime.datetime(2000, 1, 1), '2000-01-01 16:18': datetime.datetime(2000, 1, 2)},
+        },
+        'min': {
+            '1-hourly': {'2000-01-01 07:23': datetime.datetime(2000, 1, 1), '2000-01-01 16:18': datetime.datetime(2000, 1, 1)},
+            '12-hourly': {'2000-01-01 07:23': datetime.datetime(2000, 1, 1), '2000-01-01 16:18': datetime.datetime(2000, 1, 1)},
+        }
+    }
+
+    @mark.parametrize('obs_time', ['2000-01-01 07:23', '2000-01-01 16:18'])  # morning and afternoon
```

Comment:
Ah, I didn't know that you could do multiple parameters this way. That's a lot easier to read than the way that I've done it in the past. Nice.